tests/testthat/test-index.R

```diff
@@ -41,12 +41,3 @@ test_that("Omitting required template keys produces warnings", {
     "Did you forget to include config in your index string?"
   )
 })
-
-test_that("Customizing title using `name` produces a warning", {
```

Comment:
Should we raise an error now instead?

Reply:
@josegonzalez Good question. We previously raised an error in the last release, but this method was deprecated in that version, and now removed in this one.